NEWS

```diff
@@ -16,6 +16,11 @@ New features:
 * Added public factory function
   `LargeBlobRegistrationOutput.supported(boolean)`.
 * Added public factory functions to `LargeBlobAuthenticationOutput`.
+* Added `hints` property to `StartRegistrationOptions`, `StartAssertionOptions`,
+  `PublicKeyCredentialRequestOptions` and `PublicKeyCredentialAssertionOptions`,
```

Comment:
```suggestion
  `PublicKeyCredentialRequestOptions` and `PublicKeyCredentialCreationOptions`,
```
This should probably be changed

Reply:
I knew I shouldn't have written this manually :laughing: good catch, thanks!

Reply:
I force-pushed the fix, hope you don't mind.